blocks/eda-frontend/src/components/Shared/Graph.js

```diff
@@ -31,10 +31,21 @@ class Graph extends React.Component {
               const chart = this
               const series = this.series[0]
               const starttime = Date.now()
+
               function addPoints () {
                 while (!pointList.isEmpty()) {
+                  const xmax = datapoint.datapointXMax
```

Comment:
```suggestion
                  const xmax = chart.xAxis[0].max
```
Don't use datapoint. Directly get from the chart.